lib/web/routers/router.go

```diff
@@ -1,35 +1,27 @@
 package routers
 
 import (
-	"html/template"
-	"net/url"
-
 	"go-google-scraper-challenge/lib/web/controllers"
 
 	"github.com/gin-gonic/gin"
-	eztemplate "github.com/michelloworld/ez-gin-template"
 )
 
-func ComebineRoutes(engine *gin.Engine) {
-	// Register HTML renderer
-	htmlRender := eztemplate.New()
-	htmlRender.Debug = gin.IsDebugging()
-	htmlRender.TemplatesDir = "lib/web/views/"
-	htmlRender.Layout = "layouts/application"
-	htmlRender.TemplateFuncMap = template.FuncMap{
-		"isActive": isActive,
-	}
-	engine.HTMLRender = htmlRender.Init()
+var webRoutes = map[string]map[string]string{
```

Comment:
Do we need to make this variable?
Can we just use string directly?

Reply:
@hoangmirs Actually this is for the `UrlFor` template function si that I can use it on the HTML file to get the right path to the given controller and action, you can see the implementation on [the next PR](https://github.com/carryall/go-google-scraper-challenge/pull/116/files#diff-4be34e0fc44fa7d543c3ac777b0e259235377685d3ccd6f1b7b550176ebf97b1)

Reply:
I see. At first, I thought the links from a resource would be the same, so we probably won't need it.
Then I'm ok with it 👍